modules/dbfs/multi.go

```diff
@@ -0,0 +1,226 @@
+package dbfs
+
+import (
+	"strconv"
+
+	"fmt"
+
+	"github.com/CodeCollaborate/Server/modules/patching"
+	"github.com/CodeCollaborate/Server/utils"
+)
+
+// TODO (wongb): Move this to config
```

Comment:
I don't think he meant exposing it to the end user, I think he meant putting it in a config file that is just on the server that whoever manages the server can edit.